### DISCUSSION TITLE
rss feed shows episodes but will not play

### DISCUSSION THREAD

**Discussion by plum** [tramhao/termusic#306] [Q&A] [answered]
I have a particular  rss feed, https://feed.podbean.com/tntradiolive/feed.xml  which is accepted by termusic and shows all episodes in the list window,  and the list updates perfectly fine. 
However,  after sending an episode from that feed (TNT radio)  to the playlist,  when I click on an episode  to play,   termuisc player feezes, although it allows me to quit, and restart.

The same  feed url works fine in both gpodder and castero.

I wonder what's going on, in order to attempot a solution.

CHeers, John

**Comment by hasezoey** [comment, marked as answer]
please try to look into the logs and provide them.

manually running gives us the following error:
```txt
ERROR termusicplayback::rusty_backend]: Failed to play track: Custom {
    kind: InvalidInput,
    error: "error fetching https://mcdn.podbean.com/mf/web/jtekfdxajpmj5qti/Joseph_Arthur_HR4_APR289j7l2.mp3: HTTP status client error (403 Forbidden) for url (https://mcdn.podbean.com/mf/web/jtekfdxajpmj5qti/Joseph_Arthur_HR4_APR289j7l2.mp3)",
}
```

which likely means podbean does not like termusic's user-agent or something, as the same url works if just put in the browser (backend `rusty`)

EDIT: also for future reference, with #307 it would now be possible to skip to the next track, even if the enqueuing had failed for backend rusty (whereas currently / previously, once there was a enqueuing error, you couldnt do anything to make the player advance as it waited for a never coming signal)

**Reply by plum** [reply]
THanks for kind, useful answer.
